sambamba/pileup.d

```diff
@@ -197,14 +202,14 @@ struct Args {
     string[] bcftools_args;
     FileFormat input_format;
 
-    this(string[] samtools_args_, string[] bcftools_args_) {
+    this(string[] samtools_args_, bool use_bcftools, string[] bcftools_args_) {
```

Comment:
Any good reason of adding the new parameter? If it's for convenience only, it's better to redeclare it in the body as `bcftools_args_.empty`, otherwise it suggests one can provide arguments for bcftools and not use it nevertheless.